demisto_sdk/__main__.py

```diff
@@ -1,14 +1,20 @@
 # Site packages
+import sys
+
+import click
+
+try:
+    import git
+except ImportError:
+    sys.exit(click.style("Git executable cannot be found, or is invalid", fg="red"))
```

Comment:
use logger.error, then `sys.exit(1)`

Reply:
we have a bit of a problem here because of circular imports 

tools.py imports GitUtil (because of current PR which uses the GitUtil class instead of repo)
common.logger.py imports tools
Gitutil imports common.logger (in this PR)

makes circular imports :/